pkg/template/controller/templateinstance_controller.go

```diff
@@ -93,32 +109,77 @@ func (c *TemplateInstanceController) handle(templateInstance *templateapi.Templa
 		}
 	}
 
-	_, err = c.templateclient.TemplateInstances(templateInstance.Namespace).Update(templateInstance)
+	_, err = c.templateclient.TemplateInstances(templateInstance.Namespace).UpdateStatus(templateInstance)
 	if err != nil {
 		utilruntime.HandleError(fmt.Errorf("TemplateInstance update failed: %v", err))
 	}
 	return err
 }
 
+func (c *TemplateInstanceController) Run(stopCh <-chan struct{}) {
+	defer utilruntime.HandleCrash()
+	defer c.queue.ShutDown()
+
+	if !cache.WaitForCacheSync(stopCh, c.informer.HasSynced) {
+		return
+	}
+
+	go wait.Until(c.runWorker, time.Second, stopCh)
```

Comment:
most of our controllers run multiple workers, any reason for this one to only run one?